README.md

```diff
@@ -9,8 +9,9 @@ This tool aims at helping you to migrate your *PIM 1.7 standard edition* (either
 
 The 1.7 source PIM you will migrate from can be either installed locally or remotely. 
 
-The 2.0 destination PIM you will migrate to should be installed locally running on the port 80, you can install it following these [instructions](https://docs.akeneo.com/latest/install_pim/manual/system_requirements/system_requirements.html).
+The 2.0 destination PIM you will migrate to should be installed locally running on the port 80, you can install it following these [instructions](https://docs.akeneo.com/latest/install_pim/manual/index.html).
 We do not support the Docker installation yet regarding Transporteo.
+The minimum supported version is 2.0.3.
```

Comment:
I don't understand it. Do you mean you migrate from 1.7.x to 2.0.y, where x is the latest 1.7 and y >= 3 ?

Reply:
Yes